core/src/main/java/cz/o2/proxima/storage/commitlog/Partitioner.java

```diff
@@ -35,6 +35,10 @@
    */
   int getPartitionId(StreamElement element);
 
+  default int getTruncatedPartitionId(StreamElement element, int numPartitions) {
```

Comment:
We should add javadoc, if we are extending public interface.

Reply:
If we don't intend to override this I think it would be better to use the default implementation in place where we would call it. Or - for usability - we might introduce static method in `Partitioners`, there seems to be no reason for this to be placed on the interface itself.

Reply:
moved